src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -99,9 +108,15 @@ trait RawlsBillingProjectComponent {
     }
 
     def listUserProjects(rawlsUser: RawlsUserRef): ReadAction[Iterable[RawlsBillingProjectMembership]] = {
-      findProjectsByUserSubjectId(rawlsUser.userSubjectId.value).result.map { projects =>
-        projects.map { rec => RawlsBillingProjectMembership(RawlsBillingProjectName(rec.projectName), ProjectRoles.withName(rec.role)) }
+      val query = for {
+        user <- projectUsersQuery if user.userSubjectId === rawlsUser.userSubjectId.value
+        project <- rawlsBillingProjectQuery if project.projectName === user.projectName
+      } yield {
+        (user.role, project.projectName, project.status)
       }
+      query.result.map { _.map {
+        case (role, projectName, projectStatus) => RawlsBillingProjectMembership(RawlsBillingProjectName(projectName), ProjectRoles.withName(role), ProjectStatuses.withName(projectStatus))
+      }}
```

Comment:
I added the output of explain, but I'm concerned because I suspect it has no value. Since I'm changing the schema, I have to run this against my local database which has very little data. Wouldn't the plan be different depending on the size of the data? If so, I'd need a copy of the dev database, but importing that looks like it would take a very long time.